go.mod

```
@@ -2,6 +2,8 @@ module github.com/transparency-dev/witness
 
 go 1.21.0
```

```suggestion
go 1.22.7
```
Bumping the go version here, so we don't need to define the toolchain?

---

I think that would mean this module cannot be built with anything below 1.22.7 (as opposed to "you _can_ build with any of the 1.22.0 or higher tools, but we prefer 1.22.6"), which seems quite strict.

---

Sure, that's reasonable.